CHANGELOG.md

```diff
@@ -12,6 +12,136 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 1.8.0 (2023-10-18)
+
+### Process Changes
+
+- Add Change Log and the Change Log fragments process for assembling it
+  ([#1560](https://github.com/oasisprotocol/explorer/issues/1560))
+
+  This follows the same Change Log fragments process as is used by [Oasis Core].
+
+  For more details, see [Change Log fragments].
+
+  [Oasis Core]: https://github.com/oasisprotocol/oasis-core
+  [Change Log fragments]: .changelog/README.md
+
+### Features
+
+- Enable ParaTimes transfers
+  ([#1636](https://github.com/oasisprotocol/oasis-wallet-web/issues/1636))
+- Address Book
+  ([#1635](https://github.com/oasisprotocol/oasis-wallet-web/issues/1635),
+  [#1657](https://github.com/oasisprotocol/oasis-wallet-web/issues/1657),
+  [#1661](https://github.com/oasisprotocol/oasis-wallet-web/issues/1661),
+  [#1668](https://github.com/oasisprotocol/oasis-wallet-web/issues/1668),
+  [#1674](https://github.com/oasisprotocol/oasis-wallet-web/issues/1674),
+  [#1696](https://github.com/oasisprotocol/oasis-wallet-web/issues/1696))
+- Password Change
+  ([#1675](https://github.com/oasisprotocol/oasis-wallet-web/issues/1675))
+- Settings Page
+  ([#1666](https://github.com/oasisprotocol/oasis-wallet-web/issues/1666),
+  [#1630](https://github.com/oasisprotocol/oasis-wallet-web/issues/1630),
+  [#1655](https://github.com/oasisprotocol/oasis-wallet-web/issues/1655),
+  [#1713](https://github.com/oasisprotocol/oasis-wallet-web/issues/1713))
+- Delete profile modal
+  ([#1641](https://github.com/oasisprotocol/oasis-wallet-web/issues/1641))
+
+### Bug Fixes and Improvements
+
+- Export private key re-design
+  ([#1704](https://github.com/oasisprotocol/oasis-wallet-web/issues/1704),
+  [#1720](https://github.com/oasisprotocol/oasis-wallet-web/issues/1720))
+- Adjust gas cost for Sapphire and Emerald
+  ([#1712](https://github.com/oasisprotocol/oasis-wallet-web/issues/1712))
+- Don't shuffle inactive validators to the top
```

Comment:
This one seems to be a duplicate (we already have this entry for 1.7.3 release).